flytekit/clients/auth/auth_client.py

```diff
@@ -87,12 +96,28 @@ def do_GET(self):
         url = _urlparse.urlparse(self.path)
         if url.path.strip("/") == self.server.redirect_path.strip("/"):
             self.send_response(_StatusCodes.OK)
+            self.send_header("Content-type", "text/html")
             self.end_headers()
             self.handle_login(dict(_urlparse.parse_qsl(url.query)))
+            if self.server.remote_metadata.success_html is None:
+                self.wfile.write(
+                    bytes(
+                        f"""
+                <html>
+                <head>
+                    <title>Oauth2 authentication Flow</title>
+                </head>
+                <body>
+                    <h1>Log in successful to {self.server.remote_metadata.endpoint}</h1>
+                </body></html>""",
```

Comment:
nit: add `please close this window` or something like that..

Reply:
will do, i will make it a configurable thing, also add Flyte icon